sidebars.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
     ],
     "Anatomy by Feature": [
       'anatomy/environment',
+      'anatomy/github-apps',
```

Comment:
Action - Let's add this as a sub-section to Repository. Check out examples to see how i did subsections.

Reply:
Done. Created a new sub-section to Repository.